server/chat-plugins/youtube.ts

```diff
@@ -23,9 +23,11 @@ try {
 export class YoutubeInterface {
 	interval: NodeJS.Timer | null;
 	intervalTime: number;
+	updateTimer?: NodeJS.Timer;
```

Comment:
Like I said, there's no need for a timer. :[

Reply:
Well, I suppose there is, if you want to store historical data.

But also, no, blah, why are we storing historical data? Memory doesn't grow on trees!

Reply:
some historical data would be nice - maybe just havig it go back the last ~20 days would be nice?

Reply:
Capped it at 15, is that a decent solution?

Reply:
I wouldn't mind historical data in an SQLite database, but in-memory storage should be limited. A monthish could be fine in theory but it depends on just how much data there is.

Reply:
I'm gonna call it two weeks for now.